diracx-core/src/diracx/core/exceptions.py

```diff
@@ -100,3 +101,43 @@ def __init__(self, job_id, detail: str | None = None):
 
 class NotReadyError(DiracError):
     """Tried to access a value which is asynchronously loaded but not yet available."""
+
+
+class DiracFormattedError(DiracError):
```

Comment:
All exceptions apart from the 2 below inherit from `DiracError`, and this one is further customization for only a few related to pilot. What if instead you modify `DiracError`?